src/err.rs

```diff
@@ -168,13 +169,31 @@ impl PyErr {
     ///
     /// The error is cleared from the Python interpreter.
     /// If no error is set, returns a `SystemError`.
-    pub fn fetch(_: Python) -> PyErr {
+    ///
+    /// If the current error is a `PanicException` (which would have originated in other pyo3 code)
+    /// then this function will resume the uncaught panic.
```

Comment:
What does `uncaught` mean here?
We catch the panic and resume it here, right?

Reply:
Ah yes, I meant the Python exception specifically was not caught in Python code. Perhaps this can be phrased better?